desktop/core/src/desktop/js/reactComponents/FileChooser/types.ts

```diff
@@ -72,13 +72,32 @@ export interface BreadcrumbData {
   url: string;
 }
 
+interface FileStats {
```

Comment:
There's another interface Stats in the same file, with the almost the same properties. Any specific reason why that couldn't be reused?

Reply:
I had missed it. Nice catch